### PR TITLE
Fix nginx config for alert manager

### PR DESCRIPTION
**Pull request by kwangil-ha**
Signed-off-by: 하광일/TC_Core Engineering <kwangil_ha@yanolja.com>

### CODE REVIEW

File: templates/nginx-config.yaml [hunk 1]
```diff
@@ -44,9 +44,18 @@ data:
           proxy_pass      http://{{ template "cortex.fullname" . }}-distributor.{{ .Release.Namespace }}.svc.cluster.local$request_uri;
         }
 
+        location ~ /api/prom/configs/.* {
+          proxy_pass      http://cortex-configs.observability.svc.cluster.local$request_uri;
```
Comment (rsteneteg) [inline templates/nginx-config.yaml:48]: This endpoint must be dynamic, see how the other proxy_pass directives looks like

File: templates/nginx-config.yaml [hunk 2]
```diff
@@ -44,9 +44,18 @@ data:
...
+        }
+
+        location ~ /api/prom/alertmanager/.* {
+          proxy_pass      http://cortex-alertmanager.observability.svc.cluster.local$request_uri;
```
Comment (rsteneteg) [inline templates/nginx-config.yaml:52]: This endpoint must be dynamic, see how the other proxy_pass directives looks like

**Comment by khaines** [comment]
@kwangil-ha , the changes look good. Thank you for making them.

Can you please re-signoff on your latest updates so that the DCO check will pass?

File: templates/nginx-config.yaml [hunk 3]
```diff
@@ -32,28 +32,37 @@ data:
...
         proxy_read_timeout 300s;
-        
+
         location = /healthz {
           return 200 'alive';
         }
 
         location = /api/prom/push {
-          proxy_pass      http://{{ template "cortex.fullname" . }}-distributor.{{ .Release.Namespace }}.svc.cluster.local$request_uri;
+          proxy_pass      http://{{ template "cortex.fullname" . }}-distributor.{{ .Release.Namespace }}.svc.cluster.local:8080$request_uri;
```
Comment (rsteneteg) [inline templates/nginx-config.yaml:45]: Hi @kwangil-ha thanks for the update,

can we replace the hardcoded port number with `{{ .Values.config.server.http_listen_port }}`?

**Comment by khaines** [comment]
@kwangil-ha, please sign your changes so the DCO check will pass. Details of why it is failing can be found here: https://github.com/cortexproject/cortex-helm-chart/pull/44/checks?check_run_id=1254462026

**Pull request closed without merging**